common.tf

```diff
@@ -1,5 +1,8 @@
 locals {
-  az_count = local.environment == "prod" ? 2 : 1
+  az_count                      = local.environment == "prod" ? 2 : 1
+  ingest_raw_cache_bucket_name  = "${local.environment}-dr2-ingest-raw-cache"
+  pre_ingest_step_function_name = "${local.environment_title}PreIngestStepFunction"
```

Comment:
We need to agree some naming here. Preservica call `preingest` the process of forming a SIP/OPEX package, validating metadata, etc. Our Step Function in effect is doing both `preingest` and `ingest` (the Preservica bit), and may extend to post-ingest where we wait for the DR copy to be written. There's also the possibility of more complex operations upstream of this step function, e.g. validating a TDR BagIt or mapping complex assets together. I err'ed on the side of caution and called this the `ingest` step function which leaves `preingest` to the upstream process that takes a package and puts it into our DR2 BagIt format, but that's still different to Preservica's terminology.